caravel/bin/caravel

```diff
@@ -125,6 +125,34 @@ def refresh_druid():
             "[" + cluster.cluster_name + "]")
     session.commit()
 
+@manager.option(
+    '-p', '--prefix', default='data_',
```

Comment:
I think it is not good because some table in caravel db store user,permission...etc data, they don't need insert to caravel.